### PR TITLE
CalendarReader #10

### PR DESCRIPTION
**Pull request by marcelosv**
add reader for Calendar. 
Add test unit for testing CalendarReader

### CODE REVIEW

**Comment by otaviojava** [comment]
Thank you for the PR.

**Comment by otaviojava** [comment]
@marcelosv
Do you mind, create a new PR putting more test in Calendar Reader?
Basically more to methods:
When the orignal value is a String
When the original value is a Number
When the  original value is a Date

File: diana-api/src/main/java/org/apache/diana/api/reader/CalendarReader.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    public <T> T read(Class<T> clazz, Object value) {
+
+        if (Calendar.class.isInstance(value)) {
+            return (T) value;
+        }
+
+        if (Long.class.isInstance(value)) {
```
Comment (otaviojava) [inline diana-api/src/main/java/org/apache/diana/api/reader/CalendarReader.java]: Maybe fix it to look for number:

```java
if(Number.class.isInstance(value)) {
Calendar calendar = Calendar.getInstance();
 calendar.setTimeInMillis((Number) value).longValue());
...
}
```

File: diana-api/src/main/java/org/apache/diana/api/reader/CalendarReader.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            Calendar calendar = Calendar.getInstance();
+            calendar.setTimeInMillis((Long) value);
+            return (T) calendar;
+        }
+
```
Comment (otaviojava) [inline diana-api/src/main/java/org/apache/diana/api/reader/CalendarReader.java]: I miss a new condition to `Date`

```java
if (Date.class.isInstance(value)) {
...
}
```

**Comment by marcelosv** [comment]
@otaviojava 
Ok, no problem.
I will create the news PR.

File: diana-api/src/main/java/org/apache/diana/api/reader/CalendarReader.java (hunk 1 above)
Comment (danielsoro) [inline diana-api/src/main/java/org/apache/diana/api/reader/CalendarReader.java]: Good catch, should go to DateReader as well. :)
I'll update it.

**Pull request merged**